.gitignore

```diff
@@ -30,3 +30,6 @@ record*
 
 # example .res files
 tests/_results*
+
+# json reports from pytest
+.pytest_reports/*
```

Comment:
Is there a reason to hide reports? For example, `_pytest_reports` will be visible

Reply:
My reasoning was that: (a) the files in the folder are not part of the package, so one might not want to see them side-by-side with them; (b) typically, reporter tools such as coverage and similar create hidden files by default, such as `.coverage`.